### PR TITLE
A link to my github repo for my inverted index project

### PR DESCRIPTION
**Pull request by hechukwu**
#### What does PR do ?

sets up a repository for the proposed project with a detailed README and two screenshots of ESLint Airbnb style violations in the terminal and HoundCI comment on a PR

#### Description of Task to be completed?

1. Setup repository for your proposed project and at least one Pull Request 
2. two screenshots of ESLint Airbnb style violations in the terminal and HoundCI comment on a PR

#### How should this be manually tested?

This can be manually tested by visiting https://github.com/andela-hchukwu/Inverted-Index

#### Any background context you want to provide?

N/A

#### Questions:

N/A

#### Screenshots

ESLint

ESLint

HoundCI

### CODE REVIEW

**Comment by q-ode** [comment]
Kindly improve the `ReadeMe` of you repository according to the guidelines.

**Comment by hechukwu** [comment]
@q-ode please review this pull request and merge if all checks passes

**Comment by oredavids** [comment]
LGTM.

**Comment by abdulsemiu-atanda** [comment]
`External dependencies for the app` is best described with `Technologies`.

**Comment by ngozi-ekekwe** [comment]
LGTM

**Comment by ghost** [comment]
LGTM

File: 06-building-a-program-start-to-finish/setting-up-linters/output/README.md [hunk 1]
```diff
@@ -0,0 +1,9 @@
+#### Output
+- **Using**: Terminal & GitHub
+- **Deliver**: two screenshots of ESLint [Airbnb](https://github.com/airbnb/javascript)  style violations in the terminal and [HoundCI](https://houndci.com/) comment on a PR
+- **With**: *independently*
+
+#### screenshots
+![ESLint](https://cloud.githubusercontent.com/assets/25608473/22862522/b565a080-f130-11e6-8db6-88761112cdad.png)
```
Comment (q-ode) [inline 06-building-a-program-start-to-finish/setting-up-linters/output/README.md:7]: This isn't displayed in the terminal as requested.

File: 06-building-a-program-start-to-finish/setting-up-your-repository/output/README.md [hunk 2]
```diff
@@ -0,0 +1,7 @@
+#### Output
+- **Using**: GitHub
+- **Deliver**: the repository for my proposed project and at least one `Pull Request`
+- **With**: *independently*
+
+#### Github repo
+- [andela-hchukwu/Inverted-Index](https://github.com/andela-hchukwu/Inverted-Index.git)
```
Comment (q-ode) [inline 06-building-a-program-start-to-finish/setting-up-your-repository/output/README.md:7]: Nice setup!

I however can't merge this as the other output isn't done properly yet.

Kindly address!

**Comment by hechukwu** [comment]
@q-ode It is now being displayed in the terminal

**Pull request merged**